client/src/app/match-making/match-making.tsx

```diff
@@ -35,6 +37,46 @@ function MatchMaking() {
 		navigator.clipboard.writeText(gameCode)
 	}
 
+	if (status === 'private_lobby') {
+		return (
+			<MenuLayout
+				back={handleCancel}
+				title="Private Game Lobby"
+				returnText="Settings"
+				className={classNames(css.privateLobby)}
+			>
+				<div className={css.privateLobbyLeft}>
+					<p>Opponent Code</p>
+					<div className={css.code} onClick={handleCodeClick}>
+						{gameCode}
+					</div>
+					<p>Spectator Code</p>
+					<div className={css.code} onClick={handleCodeClick}>
```

Comment:
This copies the opponent code, not the spectator code.

Reply:
This has been fixed on master but needs to be merged.